app/admin/user.rb

```diff
@@ -12,6 +12,16 @@
                 :password_confirmation,
                 :is_approved
 
+  before_action :send_approval_emails, only: :update
+
+  controller do
```

Comment:
Why is it in admin ? Souldn't it be part of the user model ?
Something like `new_user_created_notification` for the RegistrationsController ? Each time the user is_approuved goes from false to true -> send email